section3/reflection.md

```diff
@@ -1,17 +1,21 @@
 ## Section 3 Reflection
 
 1. How different did your workflow feel this week, considering we asked you to follow the Pomodoro technique?
-
+`It generally felt the same. However, see question 2.`
 2. Regarding the work you did around setting intentions in Step 1 of the Pomodoro technique - how did that go? Were you surprised by anything (did you find yourself way more focused than you realized, more distracted that you thought you'd be, estimating times accurately or totally off, etc)?
-
+`I generally found myself wishing there was more time. Taking a break right at 25 minutes felt like I was relinquishing any mojo I might have had going. Also, I drink a *lot* of water, so taking regular breaks is generally good practice for me.``
```

Comment:
If, during your work time where you can define breaks, 30 or 40 minutes feels better since you are on a roll - you do you!!